### DISCUSSION TITLE
Documenting build issues for the old versions

### DISCUSSION THREAD

**Discussion by al42and** [AdaptiveCpp/AdaptiveCpp#1205] [General]
There is a problem building hipSYCL 0.9.4 + Clang 17, with an easy workaround. ACpp 23.10 works fine.

What would be the best way to document it? 

I don't expect it to get fixed, since there are no plans for more 0.9.x releases, but I think it might be helpful to have some way to make a discoverable record.

**Comment by illuhad** [comment]
Good question. We've never done something like this before. Some archive section in the readme? But that might be difficult to find or bloat the readme. And having documentation for other versions in the documentation that is bundled with the code might be confusing.
So perhaps a Wikipage? A comment on the release page? Wikipage linked from the release page? What are your preferences?

**Reply by al42and** [reply]
README, IMO, is definitely a poor solution for the reasons you described.

> So perhaps a Wikipage? A comment on the release page? Wikipage linked from the release page?

"Wikipage linked from the release page" seems to be the most straightforward. It does not bloat the release page, but should be easy to find and update.

Another way might be opening a new issue and closing it as "wontfix". Not immediately discoverable, but should be googleable.

No strong preference from me as long as one can find it and point others to it.

**Reply by al42and** [reply]
I created https://github.com/AdaptiveCpp/AdaptiveCpp/wiki/Build-instructions-for-old-versions. Feel free to edit/rename.

**Reply by illuhad** [reply]
Thanks, I think this is fine for now - if we want to document more issues beyond build instructions, perhaps we might have to restructure that page, but I don't think we need to overengineer this now :)
I have linked the wiki page on the release page: https://github.com/AdaptiveCpp/AdaptiveCpp/releases/tag/v0.9.4
